build.gradle

```diff
@@ -258,6 +278,7 @@ createTestTask('skriptTestJava8', 'Runs tests on all Java 8 environments.', envi
 createTestTask('skriptTestDev', 'Runs testing server and uses \'system.in\' for command input, stop server to finish.', environments + env, envJava, Modifiers.DEV_MODE, Modifiers.DEBUG)
 createTestTask('skriptProfile', 'Starts the testing server with JProfiler support.', environments + latestEnv, latestJava, Modifiers.PROFILE)
 createTestTask('genDocs', 'Generates the Skript documentation website html files.', environments + env, envJava, Modifiers.GEN_DOCS)
+createTestTask('genReleaseDocs', 'Generates the Skript documentation website html files for a release.', environments + env, envJava, Modifiers.GEN_RELEASE_DOCS)
```

Comment:
This should account for force generate hooks
https://github.com/SkriptLang/Skript/blob/54937c0eb89a37b99229040214edc80ccf885070/src/main/java/ch/njol/skript/test/platform/Environment.java#L251-L252